waiter/src/waiter/util/utils.clj

```diff
@@ -157,7 +163,7 @@
      :body (fn [^ServletResponse resp]
              (let [writer (OutputStreamWriter. (.getOutputStream resp))]
                (try
-                 (json/write data-map writer :value-fn stringify-elements)
+                 (json/write data-map writer :key-fn stringify-keys :value-fn stringify-elements)
```

Comment:
The tests are probably failing because `stringify-keys` doesn't exist.

Reply:
Nope, it exists—it's just not in the patch. Looks like a random failure. I restarted, and if we get a green I'll go ahead and merge this.